### ISSUE TITLE
Unable to play .m4a files due to mission ffmpeg plugin

### ISSUE THREAD

**Issue by olorton** [cmus/cmus#1258]
Since upgrading to version 6 of ffmpeg the plugins list no longer shows ffmpeg as being available. When trying to open a m4a file I get the following error.

```
Error: opening file `/Users/olorton/music-cmus/Bach; Yo‐Yo Ma/The Six Unaccompanied Cello Suites/01  - Suite No. 1 in G major, BWV 1007： I. Prélude.m4a': unsupported file format
```

This has worked fine for a few years on the same music library and only broke within the last month after upgrading cmus & ffmpeg. Cmus and ffmpeg have been installed using homebrew. I am happy to submit a pull-request to homebrew-core to fix this, if one is available, but your guidance would be appreciated.

---

1) `uname -a`:
```
Darwin FA-3346-MBPRO16 21.6.0 Darwin Kernel Version 21.6.0: Thu Mar  9 20:08:59 PST 2023; root:xnu-8020.240.18.700.8~1/RELEASE_X86_64 x86_64
```

2) `cmus --version`:
```
cmus v2.10.0
Copyright 2004-2006 Timo Hirvonen
Copyright 2008-2016 Various Authors
```

3) `cmus --plugins`:
```
Input Plugins: /usr/local/Cellar/cmus/2.10.0_2/lib/cmus/ip
  mad:
    Priority: 55
    File Types: mp3 mp2
    MIME Types: audio/mpeg audio/x-mp3 audio/x-mpeg
  opus:
    Priority: 50
    File Types: opus
    MIME Types:
  wav:
    Priority: 50
    File Types: wav
    MIME Types:
  mp4:
    Priority: 50
    File Types: mp4 m4a m4b
    MIME Types:
  cue:
    Priority: 50
    File Types:
    MIME Types: application/x-cue
  aac:
    Priority: 50
    File Types: aac
    MIME Types: audio/aac audio/aacp
  flac:
    Priority: 50
    File Types: flac fla
    MIME Types:
  vorbis:
    Priority: 50
    File Types: ogg oga ogx
    MIME Types: application/ogg audio/x-ogg

Output Plugins: /usr/local/Cellar/cmus/2.10.0_2/lib/cmus/op
  coreaudio
  ao
```

**Comment by gavtroy** [comment]
The missing FFmpeg plugin is sure to be #1251. Many distros are currently carrying a patch for this.

Incidentally there's a second issue which is that your m4a files aren't recognized by the `mp4` plugin. There's a good chance #1255 will address that.

**Comment by olorton** [comment]
Note that this is still an issue for me I'm having to compile cmus myself. Although it is fixed in master, we need a new release tag so that we can update the homebrew formula. Any chance we could do that?

**Comment by gavtroy** [comment]
@flyingmutant something to consider, the clang fix (#1269) is also not yet in a release.

**Comment by pgaskin** [comment]
Closing since the fixes have been merged (along with multiple other ffmpeg/configure-related fixes). There should be a new release soon, but due to the long period between releases, I recommend that important patches be cherry-picked by packagers where required (most are already doing this).
